<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_film_consommation</class>
 <widget class="QMainWindow" name="MainWindow_film_consommation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart cinema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>711</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Films</string>
     </attribute>
     <widget class="QLineEdit" name="recherche_film">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tab_film">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Liste Des Films :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Exporter en excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Consommations</string>
     </attribute>
     <widget class="QTableView" name="tab_cons">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Liste Des Consommation :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche_cons">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>44</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Matière</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>491</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Bénéficiaire</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>597</x>
        <y>79</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>116</y>
        <width>651</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Tapez votre message ici...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>532</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Votre mot de passe mail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Envoyer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>571</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="placeholderText">
       <string>Pièce jointe ici</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Lancer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mute">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Muter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_film"/>
   <addaction name="actionremove_film"/>
   <addaction name="actionadd_consomation"/>
   <addaction name="actionremove_consomation"/>
  </widget>
  <action name="actionadd_film">
   <property name="text">
    <string>Ajouter Un Film</string>
   </property>
  </action>
  <action name="actionremove_film">
   <property name="text">
    <string>Supprimer Un Film</string>
   </property>
  </action>
  <action name="actionadd_consomation">
   <property name="text">
    <string>Ajouter Une Consommation</string>
   </property>
  </action>
  <action name="actionremove_consomation">
   <property name="text">
    <string>Supprimer Une Consommation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
